content/en/Platform Deep Dive/Pentests/_index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Pentests"
+linkTitle: "Pentests"
+weight: 20
+description: >
+  Pentests are the cornerstone of our PtaaS platform.
+---
+
+{{% pageinfo %}}
+Launch a pentest to see the benefits of our offering in action.
+{{% /pageinfo %}}
+
+Refer to the [Getting Started](../../getting-started/) guide to set up a pentest in four steps:
```

Comment:
I much prefer stages and not steps. To me, every "click" and data entry is a "step". While we're reducing the number of steps, we still have a few.

```suggestion
Refer to the [Getting Started](../../getting-started/) guide to set up a pentest in four stages:
```